### PR TITLE
Update info.xml

### PR DESCRIPTION
**Pull request by pmaier1**
- [x] Screenshot still missing
- [x] Doc link still missing

@DeepDiver1975 Please check for correctness.

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -2,16 +2,33 @@
...
+- Increased security by shifting user authentication to an external Identity Provider
+- Seamless integration into single sign-on (SSO) environments as well as with third party products
+- Centralized client management within the Identity Provider
+- Enterprise-grade security through the use of authentication security features (e.g., multi-factor authentication) and policies (e.g., automatic token expiration on certain conditions) provided by Identity Providers
+
+The integration requires an Identity Provider which supports OpenID Connect (e.g., Keycloak, Kopano Konnect, Ping Federate, ADFS or others).
+More information on setup, configuration and migration can be found in the ownCloud Documentation.</description>
     <licence>GPLv2</licence>
     <author>Thomas Müller</author>
     <version>0.1.0</version>
```
Comment (mmattel) [inline appinfo/info.xml]: Version should be 0.2.0 according https://github.com/owncloud/openidconnect/pull/65

**Comment by mmattel** [comment]
@micbar @pmaier1 
Regarding documentation, this is the PR for it: https://github.com/owncloud/docs/pull/2110
The location, public or enterprise needs to be told to put the doc into the correct location, see comment in doc PR.

**Comment by mmattel** [comment]
As far I can see, an app image is missing.
@hurradieweltgehtunter, you did/organized that in the past...

File: appinfo/info.xml (hunk 1 above)
Comment (micbar) [reply appinfo/info.xml]: please rebase

**Comment by hurradieweltgehtunter** [comment]
Image is in progress
thanks for mentioning @mmattel

**Comment by pmaier1** [comment]
Screenshot added. Thanks to @hurradieweltgehtunter.

**Comment by mmattel** [comment]
@pmaier1 could you rebase and squash?

**Comment by pmaier1** [comment]
> @pmaier1 let me know of you need support on these topics

Please do the magic for me. THX!

**Comment by DeepDiver1975** [comment]
> Please do the magic for me. THX!

done

**Pull request merged**